### DISCUSSION TITLE
@DIR@ in services

### DISCUSSION THREAD

**Discussion by AGluk** [SoftEtherVPN/SoftEtherVPN#1949] [Q&A] [answered]
Hello everybody!
Could someone please explain what means @DIR@ in systemd services?
I can't find any useful over the internet :( 
Thanks in advance!

**Comment by Evengard** [comment, marked as answer]
This is CMake shenanigans.

Ref to https://cmake.org/cmake/help/latest/command/configure_file.html and https://github.com/SoftEtherVPN/SoftEtherVPN/blob/dc2b11918abe0566de69b7ae7bf5450cfbaa3b6d/src/CMakeLists.txt#L34

**Reply by AGluk** [reply]
Ok, thanks!

So it replaced during make by the real path.

**Reply by Evengard** [reply]
Yes, that's it.
